### PR TITLE
feat: Model Enricher

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/valueobjects/ComponentProperties.java [hunk 1]
```diff
@@ -108,7 +109,14 @@ public Optional<Integer> getOptionalInteger(String key) {
 	 */
 	public Optional<String> getOptionalPropertyFromFile(String key) {
 		String file = getProperty(key);
-		Path path = Path.of(file);
+
```
Comment (Tomvbe) [inline ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/valueobjects/ComponentProperties.java:112]: can you add a unit test for this?

File: docs/_ldio/examples/ex3-enrich-model.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+## Used Components
+
+- [Http In](../ldio-inputs/ldio-http-in)
+- [RDF Adapter](../ldio-adapters/ldio-rdf-adapter.md)
+- [Console Out](../ldio-outputs/ldio-console-out)
+- [Repository Materialiser](../ldio-outputs/ldio-repository-materialiser)
+
+## Setup
+
+For this setup, we will have two pipelines:
```
Comment (Tomvbe) [inline docs/_ldio/examples/ex3-enrich-model.md:23]: It's a pity we can't use output components in the middle of the pipeline ;)

File: docs/_ldio/examples/ex3-enrich-model.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+This can be done by mounting a rdf4j workbench image.
+
+````shell
+docker run -d -p 8081:8080 -e JAVA_OPTS="-Xms1g -Xmx4g" eclipse/rdf4j-workbench:latest
+````
+
+Once spun up, a simple repository can be configured via doing the following curl command:
+
+***test-db.ttl:***
+````text
```
Comment (Tomvbe) [inline docs/_ldio/examples/ex3-enrich-model.md:39]: These could be ```ttl as well. Currently intellij represents it the same as text but maybe when/if IDE's ever start supporting ttl out of the box, the formatting might be better.

This is just a thought, you don't have to change this.

**Review by Tomvbe — APPROVED** [review]

File: docs/_ldio/examples/ex3-enrich-model.md (hunk 2 above)
Comment (Yalz) [reply docs/_ldio/examples/ex3-enrich-model.md:23]: not happening ;)

**Pull request merged**